.gitignore

```diff
@@ -1,6 +1,14 @@
 .DS_Store
-.classpath
-.project
-.settings
-target
 test-output
+
+# Eclipse #
+/.classpath/
+/.project/
+/.settings/
+
+# Maven #
+*/target/
+
+# IntelliJ #
+/.idea/
+*.iml
```

Comment:
Oh, yeah, this _is_ a multi-module project, isn't it? I was thinking "we don't have any of _those_!"